### PR TITLE
feat(redis): Add redis init container

### PR DESCRIPTION
**Pull request by stavros-k**
**Description**

⚒️ Fixes  # 

**⚙️ Type of change**

- [x] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [x] ⚖️ My code follows the style guidelines of this project
- [x] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [x] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [x] ⬆️ I increased versions for any altered app according to semantic versioning

### CODE REVIEW

**Comment by stavros-k** [comment]
Tried some things to see how tests work, but I need to dedicate some time to read up on them first. 
Seems that vCT test isn't failing because of this PR tho

**Comment by stavros-k** [comment]
Reason of this PR is to ensure redis is active before pods that depend on it starts before them, same as we deal with the rest of  db's.

Spotted at least couple of apps that was starting before redis.

File: charts/common/templates/lib/dependencies/_redisInit.tpl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            ping
+        )
+      if [ "$response" == "PONG" ] || [ "$response" == "LOADING Redis is loading the dataset in memory" ]; then
+        LIVE=true
+        echo "$response"
+        echo "Responded!"
+      else
+        echo "$response"
+        echo "Redis not respoding... Sleeping for 2 sec..."
+        sleep 2
```
Comment (Ornias1993) [inline charts/common/templates/lib/dependencies/_redisInit.tpl:47]:
```suggestion
        sleep 10
```

Comment (Ornias1993) [inline charts/common/templates/lib/dependencies/_redisInit.tpl:47]:
```suggestion
        sleep 10
```

File: charts/common/templates/lib/dependencies/_redisInit.tpl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          timeout -s 3 2 \
+          redis-cli \
+            -h $REDIS_HOST \
+            -p $REDIS_PORT \
+            ping
+        )
+      if [ "$response" == "PONG" ] || [ "$response" == "LOADING Redis is loading the dataset in memory" ]; then
+        LIVE=true
+        echo "$response"
+        echo "Responded!"
```
Comment (Ornias1993) [inline charts/common/templates/lib/dependencies/_redisInit.tpl:43]:
```suggestion
        echo "Redis Responded, ending initcontainer and starting main container(s)..."
```

File: charts/common/Chart.yaml [hunk 3]
```diff
@@ -15,4 +15,4 @@ maintainers:
 name: common
 sources: null
 type: library
-version: 10.0.13
+version: 10.0.14
```
Comment (Ornias1993) [inline charts/common/Chart.yaml:18]:
```suggestion
version: 10.1.0
```

File: charts/common/templates/lib/dependencies/_redisInit.tpl [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            -p $REDIS_PORT \
+            ping
+        )
+      if [ "$response" == "PONG" ] || [ "$response" == "LOADING Redis is loading the dataset in memory" ]; then
+        LIVE=true
+        echo "$response"
+        echo "Redis Responded, ending initcontainer and starting main container(s)..."
+      else
+        echo "$response"
+        echo "Redis not respoding... Sleeping for 2 sec..."
```
Comment (stavros-k) [inline charts/common/templates/lib/dependencies/_redisInit.tpl:46]:
```suggestion
        echo "Redis not respoding... Sleeping for 10 sec..."
```

**Comment by Ornias1993** [comment]
I really don't see a problem adding this tbh.

**Comment by stavros-k** [comment]
Yea, it should work without problems.

**Pull request merged**